Updating conftest options

wtgee opened the pull request:
* Better config setup

wtgee committed:
Updating conftest options

* Better config setup

wtgee committed:
Quick fix to make tests run - still a problem with dither and `current_exp_num`

wtgee committed:
Merge branch 'develop' of github.com:AstroHuntsman/huntsman-pocs into update-conftest

wtgee committed:
Merge branch 'develop' of github.com:AstroHuntsman/huntsman-pocs into update-conftest

lspitler reviewed (approved):
@wtgee looks like there is a timeout issue in travis.

wtgee committed:
Merge branch 'develop' of github.com:AstroHuntsman/huntsman-pocs into update-conftest

wtgee committed:
Updating cocnftest to use fixtures ismilar to POCS

lspitler commented:
ping @wtgee on this one.

wtgee committed:
Many updates to bring in line with the POCS evening flats

* Add config items for various horizon levels
* Add flat field config items - default to evening flats
* Fix many tests

wtgee committed:
Test fixes

wtgee committed:
Fixing for tests to bring in line with POCS

wtgee committed:
* Using my repo branch for now until panoptes/POCS#541 is merged
* Adding missing state transition
* Fixing various tests

wtgee committed:
* Updating requirements
* Adding a better check for existing observation

wtgee committed:
Update default config to not take flats

wtgee committed:
Bring tests and conftest in line

Note that this doesn't include some of the flat-field items like it did previously

wtgee committed:
Long hard battle with conftest.

wtgee committed:
Removing HDR tests for now
